packages/preset-mini/src/rules/flex.ts

```diff
@@ -16,11 +16,9 @@ export const flex: Rule<Theme>[] = [
   ['flex-none', { flex: 'none' }],
 
   // shrink/grow/basis
-  [/^(?:flex-)?shrink$/, () => ({ 'flex-shrink': 1 })],
-  [/^(?:flex-)?shrink-0$/, () => ({ 'flex-shrink': 0 })],
-  [/^(?:flex-)?grow$/, () => ({ 'flex-grow': 1 })],
-  [/^(?:flex-)?grow-0$/, () => ({ 'flex-grow': 0 })],
-  [/^(?:flex-)?basis-(.+)$/, ([, d], { theme }) => ({ 'flex-basis': theme.spacing?.[d] ?? h.bracket.cssvar.auto.fraction.rem(d) })],
+  [/^(?:flex-)?shrink(?:-(.*))?$/, ([, d]) => ({ 'flex-shrink': h.bracket.number(d) ?? 1 }), { autocomplete: ['flex-shrink-<num>', 'shrink-<num>'] }],
+  [/^(?:flex-)?grow(?:-(.*))?$/, ([, d]) => ({ 'flex-grow': h.bracket.number(d) ?? 1 }), { autocomplete: ['flex-grow-<num>', 'grow-<num>'] }],
```

Comment:
I was about to submit the same fix. I'd suggest using `h.bracket.cssvar.number`.

Reply:
Ok, i will update it.